source/_docs/dns-providers/cloudflare.md

```diff
@@ -55,6 +54,31 @@ You can configure Cloudflare's CDN as an additional layer on Pantheon's Global C
 4. Remove existing redirects configured via PHP in `settings.php` or `wp-config.php`.
 5. Proceed with DNS configuration as describe in Option 1, but make sure the cloud is toggled orange, not gray.
 
+
+## CAA Records (Optional)
+
+A **CAA Record** specifies which certificate authority (**CA**) can issue HTTPS certificates for a domain.
+
+1. Select **DNS** from the Cloudflare menu bar.
+2. Select **CAA** from the dropdown menu.
+3. Enter the bare domain (`example.com`) in the **Name** field then click to configure the record value:
+
+  ![caa click to configure](/source/docs/assets/images/cf-caa.png)
+
+4. Select **Allow wildcards and specific hostnames** for the record's tag and enter  `letsencrypt.org` in the value:
```

Comment:
(optional) If you'd like to authorize other Certificate Authorities to issue certificates for your domain you can use SSLMate’s CAA Record Generator  (https://sslmate.com/caa/) to help generate a set of CAA records.

Reply:
@ari-gold What would be the purpose, if customers can only use the certificates we provide on our platform?

Reply:
They could, for example, have a subdomain serving traffic from somewhere else other than Pantheon.

Reply:
Does our recommendation to "Allow wildcards and specific hostnames" conflict with someone's ability to create records for subdomains serving traffic from somewhere else other than Pantheon?  Unless it does, I vote we leave it out to keep things simpler. 

If we add it in, folks could think they need to do something there when they don't.

Reply:
We can leave it out, but then let's find another home to generally explain CAA records